x-pack/plugins/security_solution/cypress/tasks/create_new_rule.ts

```diff
@@ -296,6 +307,10 @@ export const fillDefineThresholdRuleAndContinue = (rule: ThresholdRule) => {
   const typeThresholdField = ($el: Cypress.ObjectLike) =>
     cy.wrap($el).type(rule.thresholdField, { delay: 35 });
 
+  // cy.get('[id="indexPatterns"]').click({ force: true });
```

Comment:
we should delete all the commented out code imo

Reply:
Fixed here: 56bed04c108fc562752a4d34785fbd40702af678